### ISSUE TITLE
Is this a Mautic Plugin?

### ISSUE THREAD

**Issue by skynet** [rmuilwijk/MauticFBAdsLeadAdsBundle#1]
Can you provide additional documentation?

**Comment by rmuilwijk** [comment]
Hi Ionel,

what information are you looking for, I'd be happy to provide.

Best regards,

Raymond

**Comment by skynet** [comment]
Thanks Raymond, my question was: is this a plugin?

**Comment by rmuilwijk** [comment]
Yes you can download it and add this to the plugins directory and install it as a integration within Mautic.
